improve comment readability

sauerlandf opened the pull request:
Fix for Issue #15

sauerlandf committed:
improve comment readability

nielsranosch commented on resources/views/date/gig/listAttendances.blade.php:
```diff
@@ -49,21 +49,32 @@
+                                                                $userid = $user->id;
+                                                                $mark = '&nbsp;<i class="far fa-comment comment-toggle" id="comment-toggle-'.$userid.'-'.$gigid.'" title="'.$comment.'"></i>';
+                                                                $mark .= '<div id="comment-'.$userid.'-'.$gigid.'"" style="display: none">'.$comment.'</div>';
+                                                                $mark .= '<script type="text/javascript">$(document).ready(function(){$("#comment-toggle-'.$userid.'-'.$gigid.'").click(function(){$("#comment-'.$userid.'-'.$gigid.'").toggle();});});</script>';
```
As little JavaScript as possible in the "contents"-section and definitely no $(document).ready stuff. There is a "js"-section at the end of this file, where this can go and it already has a $(document).ready(...), where you can add your stuff to. Can use a class instead of an id and do something like:
`$('.hideable-comment').click(function() { $(this).toggle() })`

sauerlandf replied:
fixed

nielsranosch commented on resources/assets/sass/app.scss:
```diff
@@ -155,6 +155,24 @@ i.fa {
   background-color: #fff;
 }
 
+/** Make table contents wrap on mobile devices (override bootstrap) **/
+.table-responsive{
+  @media screen and (max-width: $screen-xs-max) {
+    > .table{
+      > thead,
+      > tbody,
+      > tfoot {
+        > tr {
+          > th,
+          > td {
+            white-space: normal;
+          }
+        }
+      }
+    }
+  }
+}
+
```
Does something like `.overflowing-cell { white-space:normal; }` not work?

sauerlandf replied:
override statement is now more specific

nielsranosch commented on resources/views/date/gig/listAttendances.blade.php:
```diff
@@ -49,21 +49,32 @@
+                                                            if($gig->hasCommented($user)){
+                                                                $gigid = $gig->id;
+                                                                $userid = $user->id;
+                                                                $mark = '&nbsp;<i class="far fa-comment comment-toggle" id="comment-toggle-'.$userid.'-'.$gigid.'" title="'.$comment.'"></i>';
+                                                                $mark .= '<div id="comment-'.$userid.'-'.$gigid.'"" style="display: none">'.$comment.'</div>';
+                                                                $mark .= '<script type="text/javascript">$(document).ready(function(){$("#comment-toggle-'.$userid.'-'.$gigid.'").click(function(){$("#comment-'.$userid.'-'.$gigid.'").toggle();});});</script>';
+                                                            }
+                                                            else{
+                                                                $mark = '';
+                                                            }
+                                                        ?>
                                                         @if($gig->isAttending($user) == 'yes')
-                                                            <td class="attending" title="{{ $comment }}">
+                                                            <td class="attending">
                                                                 <i class="fa fa-check"></i>{!! $mark !!}
                                                             </td>
                                                         @elseif($gig->isAttending($user) == 'maybe')
-                                                            <td class="maybe-attending" title="{{ $comment }}">
+                                                            <td class="maybe-attending">
                                                                 <i class="fa fa-question"></i>{!! $mark !!}
                                                             </td>
                                                         @elseif($gig->isAttending($user) == 'no')
-                                                            <td class="not-attending" title="{{ $comment }}">
+                                                            <td class="not-attending">
                                                                 <i class="fa fa-times"></i>{!! $mark !!}
                                                             </td>
                                                         @else
-                                                            <td class="unanswered" title="{{ $comment }}">
+                                                            <td class="unanswered">
                                                                 <i class="fa fa-minus"></i>{!! $mark !!}
                                                             </td>
                                                         @endif
```
I never liked the code-flow with $mark and it looks even worse with these changes. This is a great chance to get rid of the variable $mark and instead read "attending"/"maybe-attending"/"not-attending"/"unanswered" from a variable.
If you don't want to rework this (which is understandable), I'll do it.

sauerlandf replied:
changed it. I'm waiting for your approval

nielsranosch reviewed (changes requested):
The ideas are good and work well. Currently, the code is not in line with the rest of the project.

sauerlandf committed:
remove inline jquery scripts

sauerlandf committed:
make override more specific

sauerlandf committed:
optimize codeflow in listAttendances.blade.php

also changes behaviour of toggling

Commit:
show number of 'maybe' in GigAttendanceList

* show number of 'maybe' in GigAttendanceLise

Including speedup: Complexity of queries decreased (approx. half execution time).

* Update app.css

dummy commit to avoid unnecessary merge conflicts upon branching

* Update .gitignore

* Update .gitignore

* remove IDE artifacts

* remove IDE artifacts

sauerlandf committed:
debug calender date view call from main menu

sauerlandf committed:
add overview for reheaersal attendances

sauerlandf commented:
added a few more thigs:
- the main menu button to go to events in calendar view was broken (fixed)
- added a similar overview for rehearsal attendances (see Issue #14 )
- added the number of 'maybe' votes in these overviews (related to Issue #45 )

sauerlandf commented:
> 
> 
> added a few more thigs:
> 
>     * the main menu button to go to events in calendar view was broken (fixed)
> 
>     * added a similar overview for rehearsal attendances
> 
>     * added the number of 'maybe' votes in these overviews (relating to but not closing Issue #45 )

these overviews should be unified for events in the long run though...

sauerlandf committed:
make comments deleteable

Commit:
Merge pull request #2 from mpcg9/comment_deletion

make comments deleteable

sauerlandf commented:
Pull request #52 is included here because i forgot how to branch. Will be the last time though ;)

sauerlandf commented:
replaced by pull request #53

nielsranosch commented:
Don't worry :-)
